src/schema/mondo.yaml

```diff
@@ -1,99 +1,347 @@
 id: https://w3id.org/mondo
```

Comment:
@sabrinatoro @matentzn Changes I just made to scripting and the generated `.yml`:
1. `is_a` added
2. `exclude_children` removed
3. If a cell in the GoogleSheet is empty, there will be no field in the Enum. E.g. if there is nothing in the `notes` field for an enum, the `notes` field will not appear. Previously yesterday, it would have shown: `notes: ''`.